lib/fog/vsphere/requests/compute/host_finish_maintenance.rb

```diff
@@ -0,0 +1,16 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Real
+
```

Comment:
Sorry to get back to you so late. Unfortunately, I've been swamped. This all looks good, but I would get rid of the  empty lines before and after the methods in the "requests" classes. Other than that it looks good to me.

Reply:
removed empty lines